src/main/java/org/opentripplanner/routing/core/AStarRequestMapper.java

```diff
@@ -0,0 +1,19 @@
+package org.opentripplanner.routing.core;
+
+import org.opentripplanner.routing.api.request.RouteRequest;
+
+public class AStarRequestMapper {
+
+  public static AStarRequestBuilder map(RouteRequest opt) {
+    return AStarRequest
+      .of()
+      .setStartTime(opt.dateTime())
+      .setPreferences(opt.preferences())
+      .setArriveBy(opt.arriveBy())
+      .setWheelchair(opt.wheelchair())
+      .setParking(opt.journey().parking())
+      .setRental(opt.journey().rental())
+      .setFrom(opt.from())
+      .setTo(opt.to());
+  }
+}
```

Comment:
I like this, at some point I hope we can cleanup the package structure and put this in another package removing the dependency between the packages as well. This is a very good start.